dev.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env nix-shell
+#!nix-shell shell.nix -i bash
+
+# Installs the dependencies of all transcribee components & starts them for development purposes.
+# This script should work on a fresh clone of transcribee & should only require nix.
+
+set -euxo pipefail
+
+echo -e "\033[1m# setting up backend:\033[0m\n"
+pdm install -p backend/
+pdm run -p backend/ manage migrate
+pdm run -p backend/ manage create_superuser_if_not_exists --user test --pass test
```

Comment:
I'd vote for `admin:admin` instead of `test:test`

Reply:
maybe a combination, to signify that it is a test user?